packages/account/src/Components/text-container/__test__/text-container.spec.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { TextContainer } from '../text-container';
+
+describe('<TextContainer/>', () => {
+    it('should contain "Lorem Ipsum" in the document', () => {
+        render(
+            <TextContainer children='Lorem Ipsum'/>
+        );
+
+        expect(screen.getByText(/Lorem Ipsum/i)).toBeInTheDocument();
+    });
+
+    it('should not contain "Lorem Ipsum" in the document but <TextContainer /> exist', () => {
+        render(<TextContainer />);
+
```

Comment:
since in the description it is mentioned u are testing "lorem epsom" is  not in the document, add an assertion for it.

 expect(screen.queryByText(/Lorem Ipsum/i)).not.toBeInTheDocument();